### PR TITLE
start all indexers on new start

### PR DESCRIPTION
**Pull request by apoorvsadana**

### CODE REVIEW

**Review by EvolveArt — CHANGES_REQUESTED** [review]

File: src/handlers/indexers/indexer_types/webhook.rs [hunk 1]
```diff
@@ -67,4 +67,24 @@ impl Indexer for WebhookIndexer {
...
+        Command::new("ps")
+            // Silence  stdout and stderr
+            .stdout(Stdio::null())
+            .stderr(Stdio::null())
+            .args([
+                "-p",
+                process_id.to_string().as_str(),
+            ])
+            .spawn()
+            .expect("Could not stop the webhook indexer")
```
[the remark this answers is not in the document]
Comment (apoorvsadana) [reply src/handlers/indexers/indexer_types/webhook.rs:85]: Nope, it just shows the status of a process and it fails if the process isn't running

File: src/infra/repositories/indexer_repository.rs [hunk 2]
```diff
@@ -74,6 +79,30 @@ pub async fn get(pool: &deadpool_diesel::postgres::Pool, id: Uuid) -> Result<Ind
...
+            }
+
+            query.select(IndexerDb::as_select()).load::<IndexerDb>(conn)
+        })
+        .await
+        .map_err(adapt_infra_error)?
+        .map_err(adapt_infra_error)?;
+
+    let posts: Vec<IndexerModel> = res.into_iter().map(|post_db| adapt_indexer_db_to_indexer(post_db)).collect();
+
```
[the remark this answers is not in the document]
Comment (apoorvsadana) [reply src/infra/repositories/indexer_repository.rs:102]: Yes, that makes sense, fixed.

File: src/handlers/indexers/indexer_types/webhook.rs (hunk 1 above)
[the remark this answers is not in the document]
Comment (EvolveArt) [reply src/handlers/indexers/indexer_types/webhook.rs:85]: hm the error messages seems unclear then no?

[the remark this answers is not in the document]
Comment (apoorvsadana) [reply src/handlers/indexers/indexer_types/webhook.rs:85]: ahh right, forgot to change this

[the remark this answers is not in the document]
Comment (apoorvsadana) [reply src/handlers/indexers/indexer_types/webhook.rs]: fixed

**Review by EvolveArt — APPROVED** [review]

**Pull request merged**